### ISSUE TITLE
 Scroll is not working properly on the distance detail page, it sometimes gets stuck when scrolling up and down

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#29038]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Click on the "+" button.
2. Go to Request money > Distance.
3. Select Start and Finish points and proceed with the request.
4. Navigate to the distance detail page.
5. Attempt to scroll up and down the page
## Expected Result: 
The page should smoothly scroll up and down without getting stuck
## Actual Result: 
Scrolling is not properly, and the page sometimes gets stuck when scrolling on the distance detail page

## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.79-3
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/38435837/58401a5a-db53-48ab-b6a4-0c6c460832f0

https://github.com/Expensify/App/assets/38435837/4b2f0bf0-a008-4674-a62d-13fd4b0f12b4

**Expensify/Expensify Issue URL:**
**Issue reported by:** @ayazhussain79
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1696444461290349

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by abdel-h66** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.

Scroll is briefly blocked when trying to scroll to the distance details page, or IOU page with receipt. 

### What is the root cause of that problem?
The problem comes from the container of the `AnimatedEmptyStateBackground` used in both these pages

`src/components/ReportActionItem/MoneyRequestView.js`
`src/components/ReportActionItem/MoneyReportView.js`

```js
<View style={[StyleUtils.getReportWelcomeContainerStyle(isSmallScreenWidth), StyleUtils.getMinimumHeight(CONST.EMPTY_STATE_BACKGROUND.MONEY_REPORT.MIN_HEIGHT)]}>
   <AnimatedEmptyStateBackground />
</View>
```


### What changes do you think we should make in order to solve the problem?

I suggest removing the `StyleUtils.getMinimumHeight(CONST.EMPTY_STATE_BACKGROUND.MONEY_REPORT.MIN_HEIGHT)` as it `overridesStyleUtils.getReportWelcomeContainerStyle(props.isSmallScreenWidth) ` that is more device size friendly instead of forcing the `minHeight` to be `CONST.EMPTY_STATE_BACKGROUND.MONEY_REPORT.MIN_HEIGHT` . 

### What alternative solutions did you explore? (Optional)

The issue comes partially from the `AnimatedEmptyStateBackground` component that ignores all `pointerEvents`. Including the scroll event. somehow, setting it to auto or at least to `box-none` fixed the issue. 

```js
        <Animated.Image
            pointerEvents="box-none"
            source={EmptyStateBackgroundImage}
            style={[StyleUtils.getReportWelcomeBackgroundImageStyle(isSmallScreenWidth), animatedStyles]}
        />
```

N/A

**Comment by Arcadekwizera** [comment]
Hi, I need to be a contributor.

**Comment by Arcadekwizera** [comment]
Contributor details
Your Expensify account email: kwizeraarcade5@gmail.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01abf672ae2e36f8b1

**Comment by corpitts** [comment]
Hello, How to join to Expensify group?

**Comment by corpitts** [comment]
Contributor details
Your Expensify account email: corpitts@outlook.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01598019cf72f02b90

**Comment by corpitts** [comment]
Contributor details
Your Expensify account email: corpitts@outlook.com
Upwork Profile Link: https://www.upwork.com/freelancers/~01598019cf72f02b90

**Comment by abdel-h** [comment]
Make sure the mouse cursor is at the top of the page within around 250px. that's where the Animated background is catching all pointer events.

**Comment by abdel-h66** [comment]
Yes, I just checked on the latest main, and still could reproduce it.

**Comment by abdel-h66** [comment]
Yes I failed to mention in my comment the pattern seen ReportActionItemCreated which is how I got to my initial solution

**Comment by joekaufmanexpensify** [comment]
PR in review

**Comment by joekaufmanexpensify** [comment]
@Ollyws could you please complete the BZ checklist here so we can prep to issue payment?

**Comment by joekaufmanexpensify** [comment]
Great, TY! BZ checklist is now complete.

**Comment by joekaufmanexpensify** [comment]
All set to issue payment. @allroundexperts was assigned to the PR on 2023-10-09, and PR was merged on 2023-10-19, so no speed bonus is applicable here. This means we need to pay: 

- @allroundexperts - $500 for PR (paid via NewDot). 
- @Ollyws - $500 for C+ review (Paid via Upwork). 
- @ayazhussain79 - $50 for reporting (Paid via Upwork).

**Comment by joekaufmanexpensify** [comment]
@Ollyws offer sent for $500!

**Comment by joekaufmanexpensify** [comment]
@ayazhussain79 offer for $50 sent!

**Comment by joekaufmanexpensify** [comment]
@allroundexperts could you please request $500 via NewDot and confirm here once complete?

**Comment by ayazhussain79** [comment]
@joekaufmanexpensify offer accepted, Thank you

**Comment by joekaufmanexpensify** [comment]
Ah, okay. Thanks for flagging that! In that case, the 50% regression penalty applies here. I've adjusted the payment amounts above to reflect this.

**Comment by joekaufmanexpensify** [comment]
@Ollyws $250 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
@ayazhussain79 $50 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
Chatted with @allroundexperts and confirmed they already submitted a request here, so this is all set.

**Comment by joekaufmanexpensify** [comment]
Closing as we're good to go here. Payment summary message is [here](https://github.com/Expensify/App/issues/29038#issuecomment-1787262700). Thanks everyone!

**Comment by JmillsExpensify** [comment]
$250 payment approved for @allroundexperts based on [this comment](https://github.com/Expensify/App/issues/29038#issuecomment-1787262700).
